Makefile

```
@@ -147,10 +147,15 @@ $(GO_APP_BUILD_TARGETS): build-%:
 .PHONY: build-go
 build-go: $(GO_APP_BUILD_TARGETS)
 
+# Make sure bucketeer-httpstan is already running. If not, run "make run-httpstan".
 .PHONY: test-go
```

I was thinking of automating the test process by starting the httpstan before the tests, and stopping it after. Either way, it would be nice to have a rule to stop the container, too.

Something like this.

```suggestion
.PHONY: test-go
test-go: start-httpstan
	TZ=UTC CGO_ENABLED=0 go test -v ./pkg/...
	make stop-httpstan
```

---

Add stop-httpstan. Manually run command before make test-go.